examples/metrics/src/main/java/io/opentelemetry/example/metrics/ObserverMeterExample.java

```diff
@@ -0,0 +1,29 @@
+package io.opentelemetry.example.metrics;
+
+import io.opentelemetry.OpenTelemetry;
+import io.opentelemetry.metrics.LongValueObserver;
+import io.opentelemetry.metrics.LongValueObserver.ResultLongValueObserver;
+import io.opentelemetry.metrics.Meter;
+
+/**
+ * In this example we are setting callback to get asynchronously jvm memory usage
```

Comment:
As above, the javadoc should explain what we're trying to teach people with the example.